src/main/java/com/cloudbees/jenkins/plugins/bitbucket/hooks/PullRequestHookProcessor.java

```diff
@@ -25,33 +25,40 @@
 
```

Comment:
Looks to me like a significant enough change to merit a bump in the copyright's year.
I might have missed that in other files, just realized it now.